### DISCUSSION TITLE
When will the mobile flet app be updated to SDK 0.25 ?

### DISCUSSION THREAD

**Discussion by 0Liposo0** [flet-dev/flet#4561] [Q&A]
The flet mobile application is very useful for testing when I develop for mobile, but I am already using SDK 0.25, while the Apple store only has 0.24.1, any prediction on when it will be updated ?

**Comment by ndonkoHenri** [comment]
In the coming days. No ETA, sorry.
Sub here: https://github.com/flet-dev/flet/issues/4539
